### DISCUSSION TITLE
Wyze required versions

### DISCUSSION THREAD

**Discussion by fotingo** [gtxaspec/wz_mini_hacks#767] [Q&A] [answered]
Hello,
I would like to install this mini hacks, but the required Wyze firmware versions are not available on their site anymore.
Are those really necessary?
If yes, where can I find those?
4.36.9.131
4.36.9.139 

Thanks.

**Comment by endertable** [comment]
Try this:

`https://s3-us-west-2.amazonaws.com/wuv2/upgrade/WYZE_CAKP2JFUS/firmware/4.36.9.139.tar`

**Comment by fotingo** [comment]
Awesome, thank you.
Since my Wyze cam v3 has v4.6.x, I think the instructions I saw said I need to downgrade to 4.36.9.131, then flash 4.36.9.139.. is that not so?
or can I just flash this version only?

**Reply by fotingo** [reply, marked as answer]
Oh wait..
I found a site that has older rtsp versions in case someone else needs them.
https://leo.leung.xyz/wiki/Wyze_Cam

**Comment by fotingo** [comment]
Did the install, but when trying to access the camera using a web browser, I get this..

Wyze Mini Web Server Menu
wz_mini.conf is missing the line WEB_SERVER_OPTIONS
to populate it with full capabilities:

WEB_SERVER_OPTIONS="cam config car jpeg multicam diag status"
